content/en/docs/concepts/extend-kubernetes/api-extension/_index.md

```diff
@@ -2,3 +2,18 @@
 title: Extending the Kubernetes API
 weight: 30
 ---
+
+Custom resources are extensions of the Kubernetes API. Kubernetes provides two ways to add custom resources to your cluster:
+
+- The [CustomResourceDefinition](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+  (CRD) mechanism allows you to declaratively define a new custom API with an API group, kind, and
+  schema that you specify.
+  The Kubernetes control plane serves and handles the storage of your custom resource. CRDs allow you to
+  create new types of resources for your cluster without writing and running a custom API server. 
+- The [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/)
+  sits behind the primary API server, which acts as a proxy.
+  This arrangement is called API Aggregation(AA), which allows you to provide
```

Comment:
Nit if you do an update - please add a space between "Aggregation" and the opening "(".